update-dependencies/DockerfileShaUpdater.cs

```diff
@@ -29,11 +29,11 @@ public DockerfileShaUpdater(string dockerfilePath) : base()
         }
 
         protected override string TryGetDesiredValue(
-            IEnumerable<DependencyBuildInfo> dependencyBuildInfos,
-            out IEnumerable<BuildInfo> usedBuildInfos)
+            IEnumerable<IDependencyInfo> dependencyBuildInfos,
```

Comment:
`using Microsoft.DotNet.VersionTools;` that will not be required now.

Reply:
Removed